core/stream/convert.go

```diff
@@ -373,13 +373,34 @@ func ftou(count int, dst, src buf) error {
 	dstBits, srcBits := dstTy.Bits(), srcTy.Bits()
 	dstMask := (1 << dstBits) - 1
 	ss := binary.BitStream{Data: src.bytes, ReadPos: src.offset}
+	ss2 := binary.BitStream{Data: src.bytes, ReadPos: src.offset}
```

Comment:
Please use a bit more semantic variable naming. What's the 2 here? Maybe you can improve the other variable name "ss"

Reply:
I'll do better and get rid of it.

Reply:
It stands for source stream, btw